infrastructure/core/main.tf

```diff
@@ -48,6 +48,20 @@ resource "azurerm_storage_account" "core" {
   }
 }
 
+resource "azurerm_private_dns_zone" "blobcore" {
```

Comment:
Hehe you're not wrong - I think it should stay just for that reason ;) think we keep "blob" in the name because it's a private link just for the blob service, not storage in general